newsroom/agenda/views.py

```diff
@@ -95,11 +100,16 @@ def item(_id):
             contacts=get_public_contacts(item),
             links=get_links(item),
             is_admin=is_admin_or_internal(user),
+            user_profile_data=user_profile_data,
         )
 
-    data = get_view_data()
+    data = await get_view_data()
     data["item"] = item
-    return flask.render_template("agenda_index.html", data=data, title=item.get("name", item.get("headline")))
+    return flask.render_template(
+        "agenda_index.html",
+        data=data,
+        title=item.get("name", item.get("headline"), user_profile_data=user_profile_data),
+    )
```

Comment:
There is an error here in pytests:  `dict.get() takes no keyword arguments`. I'm guessing this keyword argument was meant for the `render_template` call instead?

Reply:
Yes, you're right. Thanks for pointing that out!